Assets/Scripts/Player/CollisionDetection.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using UnityEngine;
+
+/*
+ * TODO This can probably be removed and added to some sort of player object when we have that.
+ */
+namespace Player
+{
+    public class CollisionDetection : MonoBehaviour
+    {
+        private void OnTriggerEnter2D(Collider2D other)
+        {
+            if (other.gameObject.layer == 10)
```

Comment:
I've added several new layers to the game. Layer 8 is for the player. Layer 9 is the shield. Layer 10 is for bullets. This helps us determine what kind of object we're interacting with.

Reply:
Can you add this as a comment in the code somewhere?

Reply:
Added a README to the scripts directory.